tutorials/hist/h1readanddraw.C

```diff
@@ -33,6 +34,7 @@ void h1draw()
    if (!example) return;
 
    example->ls();
+   // Get the histogram from the ROOT File
    TH1 *hpx = (TH1*)example->Get("hpx");
```

Comment:
Can you use the interface I was [proposing earlier](https://github.com/root-project/root/pull/8699#discussion_r672285210), please, including the comment I proposed?

Reply:
Okay, I'll edit it.